.github/workflows/main-workflow.yml

```diff
@@ -17,4 +17,7 @@ jobs:
         run:  sudo npm install
       
       - name: BMS monitor
-        run: node bms-monitor.js
+        run: "node bms-monitor.js"

```

Comment:
Purpose of running bms-monitor.js?
It has no code in global scope, other than the export. So nothing will execute?